README.md

```diff
@@ -104,41 +94,82 @@ The only major differences are :
 
 ### ⛓️ Chainable methods
 
-**uniq**: deduplicates records from your CSV file. Rows are deduplicated by the result of the provided callback, or by the value of all columns.
+**map(callback)**: maps each record to a new shape. The output will be typed accordingly.
+
+**pick(keys)**: picks a subset of properties from the records. `keys` is described in the [lodash.pick](https://lodash.com/docs/#pick) documentation.
 
-**filter**: filters out records.
+**filter(callback)**: filters out records.
 
-**map**: maps each record to a new shape. The output will be automatically typed correctly.
+**forEach(callback)**: this is executed on each record, but doesn't alter the data of the rows.
 
-**forEach**: this is executed on each record, but doesn't actually alter the output data.
+**uniq(iterator)**: deduplicates records from your CSV file. It can accept as argument :
+
+* A column name to deduplicate on that column
+* An array of column anmes to deduplicate on the combination of those columns
+* A callback returning a string to deduplicate on the value of that string
 
 ### 🚧 Terminator methods
 
-**find**: returns the first matching record
+_The following methods only ever consume one row at a time so they are safe to use on very large files_
 
-**every**: returns `true` if all records match
+**process**: Executes the pipeline on all the rows, but without outputting any data. This is useful for example when your pipeline is based on `forEach` and you want to discard the final output data.
 
-**some**: returns `true` if at least one record matches
+**find(callback)**: returns the first matching record
 
-**includes**: returns `true` if the final output contains the value passed. This uses [`lodash.isEqual`](https://lodash.com/docs/4.17.15#isEqual) so the value can be a primitive, object, array, etc...
+**findIndex(callback)**: returns the index of the first matching record
 
-```javascript
-await withCSV('my.csv')
-  .query(['phone'])
-  .map(row => row.phone)
-  .includes('0612345678')
-  // true
-```
+**every(callback)**: returns `true` if all records match
 
-**process**: Executes the pipeline on all the rows, but without outputting any data. This is useful for example when your pipeline is based on `forEach` and you want to discard the final output data.
+**some(callback)**: returns `true` if at least one record matches
+
+**includes(value)**: returns `true` if the final output contains the value passed. This uses [`lodash.isEqual`](https://lodash.com/docs/4.17.15#isEqual) so the value can be a primitive, object, array, etc...
```

Comment:
nit: Array.includes fait un Object.is (modulo pour comparer NaN avec Nan) pour comparer. On ne veut pas utiliser un autre nom pour cet opérateur? Comme propositions :
- includesDeepEqual
- someValueEquals
- someValue